lib/resolvers/mutation.js

```diff
@@ -72,10 +72,10 @@ function throwUnimplemented(info) {
  * handler.
  *
  * @function resolveMutation
- * @param {Object} [obj]
  * @param {Object} args
  * @param {Object} context
  * @param {Object} info
+ * @param {Object} typeName
```

Comment:
Quick note here: `typeName` should be a string. My bad in the first place to leave it out.

Reply:
Updated to String